components/card/arrow/src/index.scss

```diff
@@ -0,0 +1,51 @@
+/* sass-lint:disable no-important*/
+@import '~@schibstedspain/sui-theme/lib/settings-compat-v7/index';
```

Comment:
Not necessary, i think you can delete it.

Reply:
In order to use the @include sui-card, I need @import '~@schibstedspain/sui-theme/lib/settings-compat-v7/index';

Reply:
are you sure? 
this mixin isn't located in the `settings-compat` https://github.com/SUI-Components/sui-theme/blob/master/src/components/_card.scss#L6

Reply:
Some of the variables used by sui-card are defined in settings-compat-v7. 🤷‍♂️

Reply:
ok, i see. This means that non deprecated components are using a deprecated config file.

Same happens with buttons
https://github.com/SUI-Components/sui-theme/blob/master/src/components/_button.scss#L1
Which is using a deprecated file
https://github.com/SUI-Components/sui-theme/blob/master/src/settings-compat-v7/_components.scss#L81

and some other components.

is this something that we expect @jcger ?
should we move this vars outside the `compat` folder?

anyways, this is not related with this PR, IMHO we can manage this in a further PR.

Reply:
Yep, it's tech debt we will face once we end developing all of our atoms and molecules